cmd/ak/cmd/runtimes/run.go

```diff
@@ -135,7 +135,7 @@ func run(ctx context.Context, b *sdkbuildfile.BuildFile, path string) (map[strin
 		defer cancel()
 	}
 
-	run, err := runtimes().Run(ctx, sdktypes.NewRunID(), path, b, nil, cbs)
+	run, err := runtimes().Run(ctx, sdktypes.NewRunID(), sdktypes.InvalidSessionID, path, b, nil, cbs)
```

Comment:
Will this generate a new valid session ID? And will the runtime pass it to Python?

Reply:
No real sessionID for cli call for run. It's like a manual run. So we don't really care about session (there is no session)

Reply:
OK